src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -0,0 +1,2201 @@
+				OHServiceExceptionUtil.showMessages(e);
+			}
+			if (inventory != null) {
+				supplierCombo.setSelectedItem(supplierSelected);
+				supplier = supplierSelected;
+			}
+			supplierCombo.addActionListener(actionEvent -> {
+				supplier = (Supplier) supplierCombo.getSelectedItem();
+			});
+		}
+		return supplierCombo;
+	}
+
+	private JComboBox<Ward> getJComboDestination() {
+		Ward destinationSelected = null;
+		if (destinationCombo == null) {
+			destinationCombo = new JComboBox<Ward>();
+			try {
+				List<Ward> wards = wardManager.getWards();
+				destinationCombo.addItem(null);
+				for (Ward ward : wards) {
+					destinationCombo.addItem(ward);
+					if (inventory != null && ward.getCode().equals(inventory.getDestination())) {
+						destinationSelected = ward;
+					}
+				}
+			} catch (OHServiceException e) {
+				OHServiceExceptionUtil.showMessages(e);
+			}
+			if (inventory != null) {
+				destinationCombo.setSelectedItem(destinationSelected);
+				destination = destinationSelected;
+			}
+			destinationCombo.addActionListener(actionEvent -> {
+				destination = (Ward) destinationCombo.getSelectedItem();
+			});
+		}
+		return destinationCombo;
+	}
+
+	private int getPosition(MedicalInventoryRow inventoryRow) {
+		int position = -1;
+		int i = 0;
+		for (MedicalInventoryRow invR : inventoryRowSearchList) {
+			if (invR.getMedical().getCode().equals(inventoryRow.getMedical().getCode())) {
+				position = i;
+			}
+			i = i + 1;
+		}
+		return position;
+	}
+
+	private boolean existInInventorySearchList(MedicalInventoryRow inventoryRow) {
+		boolean found = false;
+		List<MedicalInventoryRow> invRows = inventoryRowSearchList.stream()
+			.filter(inv -> inv.getMedical().getCode().equals(inventoryRow.getMedical().getCode())).collect(Collectors.toList());
+		if (!invRows.isEmpty()) {
+			for (MedicalInventoryRow invR : invRows) {
+				if (inventoryRow.getLot() != null && invR.getLot() != null) {
+					if (inventoryRow.getLot().getCode().equals(invR.getLot().getCode())) {
+						found = true;
+						break;
+					}
+				} else {
+					if (invR.getLot() == null && inventoryRow.getLot() == null) {
+						found = true;
+						break;
+					}
+				}
+			}
+		}
+
+		return found;
+	}
+
+	private void addMedInRowInInventorySearchList(MedicalInventoryRow inventoryRow) {
+		int position = getPosition(inventoryRow);
+		if (position == -1) {
+			position = inventoryRowSearchList.size();
+			inventoryRowSearchList.add(position, inventoryRow);
+		} else {
+			inventoryRowSearchList.add(position + 1, inventoryRow);
+		}
+		if (inventoryRow.getId() == 0) {
+			inventoryRowListAdded.add(inventoryRow);
+		}
+	}
+
+	private void resetVariables() {
+		inventoryRowsToDelete.clear();
+		lotsDeleted.clear();
+		inventoryRowListAdded.clear();
+		lotsSaved.clear();
+	}
+
+	private boolean checkParametersChanges(String wardCode, String chargeCode, String dischargeCode, Integer supplierId, String reference, LocalDateTime date) {
```

Comment:
the parameter `String reference` is not used in the method; it should be removed

Reply:
it should... fixed. Thanks!